tiled/client/dataframe.py

```diff
@@ -160,22 +160,19 @@ def read(self, columns=None):
         The result will be internally partitioned with dask.
         """
         structure = self.structure()
+        meta = structure.meta
+        if columns is not None:
+            meta = meta[columns]
+
         # Build a client-side dask dataframe whose partitions pull from a
         # server-side dask array.
         name = f"remote-dask-dataframe-{self.item['links']['self']}"
         dask_tasks = {
             (name, partition): (self._get_partition, partition, columns)
             for partition in range(structure.npartitions)
         }
-        meta = structure.meta
-
-        if columns is not None:
-            meta = meta[columns]
-        ddf = dask.dataframe.core.DataFrame(
-            dask_tasks,
-            name=name,
-            meta=meta,
-            divisions=(None,) * (1 + structure.npartitions),
+        ddf = dask_expr.DataFrame.from_map(
+            dask_tasks, meta=meta, divisions=(None,) * (1 + structure.npartitions)
```

Comment:
I think you want:

```
dd.from_map(self._get_partition, range(structure.npartitions), columns=columns)
```

Please **do not**. import dask_expr directly. You should import dask.dataframe as dd and use whatever is in the namespace. dask-expr was merged into dask.dataframe and will go away now with the latest release